packages/CodingStandard/src/Fixer/Php/ClassStringToClassConstantFixer.php

```diff
@@ -0,0 +1,116 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\CodingStandard\Fixer\Php;
+
+use PhpCsFixer\Fixer\DefinedFixerInterface;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+use SplFileInfo;
+
+final class ClassStringToClassConstantFixer implements DefinedFixerInterface
+{
+    /**
+     * @var string
+     */
+    private const CLASS_OR_INTERFACE_PATTERN = '#^[A-Z]\w*[a-z]\w*(\\\\[A-Z]\w*[a-z]\w*)+\z#';
```

Comment:
drop anything that may creates wrong changes, it's better to convert too few than too much